For whatever reason they don't use pkg-config for this.
We will substitute this as needed from the template.

--- a/meson.build
+++ b/meson.build
@@ -138,7 +138,8 @@ if libunwind_dep.found()
      dependencies: [
        valac.find_library('libunwind', dirs: [vapi_dir, unwind_lib]),
        cc.find_library('libunwind', dirs: unwind_lib),
-       cc.find_library('libunwind-generic', dirs: unwind_lib)
+       cc.find_library('libunwind-generic', dirs: unwind_lib),
+       @UCONTEXT_LIB@
      ],
    )
